packages/shared/src/components/feeds/FeedSettings/FeedSettingsCreate.tsx

```diff
@@ -0,0 +1,256 @@
+import React, { ReactElement, useEffect, useRef, useState } from 'react';
+import { useRouter } from 'next/router';
+import classNames from 'classnames';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { ClientError } from 'graphql-request';
+import { Modal } from '../../modals/common/Modal';
+
+import {
+  CreateFeedProps,
+  useActions,
+  useFeeds,
+  usePlusSubscription,
+  useProgressAnimation,
+  useToastNotification,
+} from '../../../hooks';
+import { emojiOptions, webappUrl } from '../../../lib/constants';
+import { Button, ButtonProps } from '../../buttons/Button';
+import { ButtonSize, ButtonVariant } from '../../buttons/common';
+import { TextField } from '../../fields/TextField';
+import { IconSize } from '../../Icon';
+import { HashtagIcon } from '../../icons';
+import {
+  Typography,
+  TypographyType,
+  TypographyTag,
+} from '../../typography/Typography';
+import { LogEvent } from '../../../lib/log';
+import { useLogContext } from '../../../contexts/LogContext';
+import { labels } from '../../../lib/labels';
+import { getFeedSettingsQueryKey } from '../../../hooks/useFeedSettings';
+import { useAuthContext } from '../../../contexts/AuthContext';
+import { PlusUser } from '../../PlusUser';
+import { ModalClose } from '../../modals/common/ModalClose';
+import { ActionType } from '../../../graphql/actions';
+
+const CreateTitle = ({ className }: { className?: string }): ReactElement => {
```

Comment:
This really triggers me XD better just plain title with additional renderprop, but let's go for now.
I might refactor it in break

Reply:
removed